GNUmakefile

```
@@ -217,6 +217,10 @@ proto:
 		protoc -I . -I ../../.. --go_out=plugins=grpc:. $$file; \
 	done
 
+.PHONY: generate-examples
+generate-examples: command/job_init.bindata_assetfs.go
```

```suggestion
generate-examples: command/job_init.bindata_assetfs.go

```

Would be nice to just give ourselves a lil bit of whitespace here

---

Sure thing.